packit_service/worker/checker/vm_image.py

```diff
@@ -0,0 +1,97 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import logging
+
+from packit_service.models import CoprBuildTargetModel, BuildStatus
+from packit_service.worker.checker.abstract import Checker
+from packit_service.worker.mixin import (
+    GetVMImageDataMixin,
+    GetReporterFromJobHelperMixin,
+)
+from packit_service.models import (
+    VMImageBuildStatus,
+)
+
+from packit_service.worker.reporting import BaseCommitStatus
+
+from packit_service.constants import DOCS_VM_IMAGE_BUILD
+
+logger = logging.getLogger(__name__)
+
+
+class GetVMImageBuildReporterFromJobHelperMixin(
+    GetReporterFromJobHelperMixin, GetVMImageDataMixin
+):
+    status_name = "vm-image-build"
+
+    def get_build_check_name(self) -> str:
+        if self.identifier:
+            return f"{self.status_name}-{self.chroot}-{self.identifier}"
+        else:
+            return f"{self.status_name}-{self.chroot}"
+
+    def report_pre_check_failure(self, markdown_content):
+        self.report(
+            state=BaseCommitStatus.neutral,
+            description="Check for VM Image build job failed",
```

Comment:
```suggestion
            description="VM Image Build job failed internal checks",
```
this is a tiny bit more descriptive to me